### DISCUSSION TITLE
Getting the same color theme as on the Install page

### DISCUSSION THREAD

**Discussion by aagha** [black7375/Firefox-UI-Fix#878] [Q&A] [answered]
Is there a guide for getting the same color themes shown on the Install page?

https://github.com/black7375/Firefox-UI-Fix?tab=readme-ov-file#introduction

My default install looks like this:

![image](https://github.com/black7375/Firefox-UI-Fix/assets/5522103/9438c33d-1d7c-47d0-8154-d760ccfa835e)

I have Lepton-Proton-Style.zip - 8.6.0 installed.

**Comment by black7375** [comment, marked as answer]
That's a just bundled dark theme!!
![image](https://github.com/black7375/Firefox-UI-Fix/assets/25581533/3348ba57-30d2-4e07-b42c-c369cc1c8f50)

**Reply by aagha** [reply]
Well wow. Thanks! :-)
